tm2py/controller.py

```diff
@@ -374,43 +374,24 @@ def _calculate_num_processors(self, cpu_processors: int):
     
     def warmstart_check(self):
         """
-        Check if required input files and folders exist before running the components.
+        Check if required warmstart demand files and folders exist before running the components.
```

Comment:
I recommend keeping the warmstart demand files separate from the demand files that get generated and overwritten by global iteration, treat it as a static input.

Reply:
There's a new section in the config that defines the warmstart input locations. They will be copied over to the highway & transit demand file locations per the rest of the config.